matplotcheck/tests/conftest.py

```diff
@@ -25,6 +26,27 @@ def pd_df_timeseries():
     )
 
 
+@pytest.fixture
+def pd_gdf():
+    """Create a geopandas GeoDataFrame for testing"""
+    df = pd.DataFrame(
+        {
+            "lat": np.random.randint(-85, 85, size=100),
+            "lon": np.random.randint(-180, 180, size=100),
+        }
+    )
+    gdf = gpd.GeoDataFrame(
+        {"A": np.arange(100)}, geometry=gpd.points_from_xy(df.lon, df.lat)
+    )
+
+    return gdf
+
+
+@pytest.fixture
```

Comment:
@ryla5068 will you please add docstrings to all fixtures so we know what they are for. it can be a short docstring (single line)